app/(auth)/signin/page.tsx

```diff
@@ -57,32 +66,109 @@ export default function SignIn() {
-            {errors.email && (
-              <FormErrorMessage>{errors.email.message}</FormErrorMessage>
-            )}
-          </FormControl>
-          <FormControl isInvalid={!!errors.password}>
-            <FormLabel>Password</FormLabel>
-            <Input type="password" {...register('password')} />
-            {errors.password && (
-              <FormErrorMessage>{errors.password.message}</FormErrorMessage>
-            )}
-          </FormControl>
-          <PrimaryButton isLoading={isLoading} type={'submit'}>
-            Sign In
-          </PrimaryButton>
-          <Heading>{"Don't"} have an account yet?</Heading>
-          <Link href="/signup/email">Sign Up</Link>
-        </Flex>
-      </Box>
-    </>
+    <Box
+      as="main"
+      h="100vh"
+      w="100vw"
+      bg={{
+        base: 'white',
+        md: 'linear-gradient(242.77deg, #C4E6E6 1.15%, #085783 100%)'
+      }}
+      color="black"
+    >
+      <AbsoluteCenter>
+        <Card
+          bg="#F8F8F8"
+          boxShadow={{ base: '0', md: '0 4px 16px rgba(0, 0, 0, 0.1)' }}
+          maxW={{ base: '100%', lg: 'container.sm' }}
+          py={{ base: '40px', md: '40px' }}
+          px={{ base: '16px', md: '80px' }}
+        >
+          <CardHeader w="100%" p="0">
+            <Center w="100%">
+              <Image src={logo} width={144} height={40} alt="Tabi Memo" />
+            </Center>
+            <VStack
+              w="100%"
+              mt={{ base: '28px', md: '40px' }}
+              align={{ base: 'center', md: 'start' }}
+            >
+              <Heading
+                as="h1"
+                fontSize={{ base: '2xl', md: '4xl' }}
+                color="gray.700"
+              >
+                Welcome Back!
+              </Heading>
+              <Heading
+                as="h2"
+                fontSize="sm"
+                color="gray.700"
+                fontWeight="regular"
+              >
+                Sign in to your account
+              </Heading>
+            </VStack>
+          </CardHeader>
+          <CardBody
+            as="form"
+            onSubmit={signInHandler}
+            mt={{ base: '38px', md: '42px' }}
+            gap={{ base: '30px', md: '40px' }}
+            p="0%"
+            w={{ base: '80vw', md: 'unset' }}
+            maxW="500px"
+          >
+            <VStack gap="24px">
+              <FormControl isInvalid={!!errors.email}>
+                <FormLabel>Email</FormLabel>
+                <InputForm
+                  {...register('email')}
+                  minW={{ base: '300px', md: '380px' }}
+                />
+                {errors.email && (
+                  <FormErrorMessage>{errors.email.message}</FormErrorMessage>
+                )}
+              </FormControl>
+              <FormControl isInvalid={!!errors.password}>
+                <FormLabel>Password</FormLabel>
+                <InputForm type="password" {...register('password')} />
```

Comment:
Can you add the icon to show or hide the password like the Figma?

Reply:
I added it!